lighthouse-cli/chrome-launcher.js

```diff
@@ -0,0 +1,187 @@
+'use strict';
+
+const childProcess = require('child_process');
+const fs = require('fs');
+const path = require('path');
+const mkdirp = require('mkdirp');
+const inquirer = require('inquirer');
+const net = require('net');
+const rimraf = require('rimraf');
+
+const spawn = childProcess.spawn;
+const execSync = childProcess.execSync;
+
+const platformError = new Error('Platform ' + process.platform + ' is not supported');
+
+function defaults(val, def) {
+  return typeof val === 'undefined' ? def : val;
+}
+
+module.exports = class Launcher {
+  constructor(opts) {
+    opts = opts || {};
+    // choose the first one (default)
+    this.head = defaults(opts.head, true);
+
+    this.pollInterval = 500;
+
+    this.chromeInstances = [];
```

Comment:
ideally yes... I faced situations where chrome wasn't killed properly, and lighthouse wasn't able to use debugging port and every time I ran lighthouse it kept spawning new chrome instances. so I kept track of all of them.. try it out removing this, if you face this too, lets find why chrome isn't killed properly.